contrib/checkstyle/src/mill/contrib/checkstyle/CheckstyleModule.scala

```diff
@@ -0,0 +1,184 @@
+package mill.contrib.checkstyle
+
+import com.etsy.sbt.checkstyle.CheckstyleSeverityLevel.CheckstyleSeverityLevel
+import com.etsy.sbt.checkstyle.{
+  CheckstyleConfigLocation,
+  CheckstyleSeverityLevel,
+  CheckstyleXSLTSettings
+}
+import mill.api.Logger
+import mill.{Agg, T}
+import mill.scalalib.{Dep, DepSyntax, JavaModule}
+import mill.util.Jvm
+import net.sf.saxon.s9api.Processor
```

Comment:
I'm curious why we need this external dependency for `Processor`? What is it doing that we cannot do via `scala.xml` or `javax.xml`?

Reply:
Just took the impl from the SBT plugin, I don't have a particular preference myself.